app/helpers/blacklight/blacklight_helper_behavior.rb

```diff
@@ -12,7 +12,10 @@ module Blacklight::BlacklightHelperBehavior
   #
   # @return [String] the application name
   def application_name
-    t('blacklight.application_name')
+    return t('blacklight.application_name') if I18n.locale == :en
```

Comment:
Is there something special about this line that' functionality different than the line below?